lib/iris/tests/unit/common/lenient/test__qualname.py

```diff
@@ -4,39 +4,34 @@
 # See LICENSE in the root of the repository for full licensing details.
 """Unit tests for the :func:`iris.common.lenient._qualname`."""
 
-# Import iris.tests first so that some things can be initialised before
-# importing anything else.
-import iris.tests as tests  # isort:skip
-
 from inspect import getmodule
-from unittest.mock import sentinel
 
 from iris.common.lenient import _qualname
 
 
-class Test(tests.IrisTest):
-    def setUp(self):
+class Test:
+    def setup_method(self):
```

Comment:
I only just realised that this **_does_** work, and why pytestify does it this way.
The docs are rather quiet about it, but it is covered [here](https://docs.pytest.org/en/7.1.x/how-to/xunit_setup.html?highlight=setup_method#method-and-function-level-setup-teardown)

Nevertheless., I think I still prefer `@fixture(autouse=True)`, and maybe we should standardise on that ??
In particular, I think a "setup_method" is **_not_** a fixture, and is not called in quite the same way, and it can't reference another fixture, e.g. mocker for a 'mocker.patch'.
```suggestion
    @pytest.fixture(auto_use=True)
    def _setup(self):
```

Reply:
Yup, all true.

Perhaps worth a sprint discussion?

I've erred on the side of using a `fixture(autouse=True)` when I don't need to use the `mocker` fixture in the `setup`, otherwise default to the `pytest` convenience of `setup_method`, which like you mention isn't a fixture.

I'm happy either way